ibm/resource_ibm_is_instance_template.go

```diff
@@ -292,54 +313,38 @@ func resourceIBMISInstanceTemplate() *schema.Resource {
 				},
 			},
 
-			isPlacementTargetDedicatedHost: {
-				Type:          schema.TypeString,
-				Optional:      true,
-				ForceNew:      true,
-				ConflictsWith: []string{isPlacementTargetDedicatedHostGroup},
-				Description:   "Unique Identifier of the Dedicated Host where the instance will be placed",
-			},
-
-			isPlacementTargetDedicatedHostGroup: {
-				Type:          schema.TypeString,
-				Optional:      true,
-				ForceNew:      true,
-				ConflictsWith: []string{isPlacementTargetDedicatedHost},
-				Description:   "Unique Identifier of the Dedicated Host Group where the instance will be placed",
+			isInstanceTemplateResourceGroup: {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Optional:    true,
+				Computed:    true,
+				Description: "Instance template resource group",
 			},
 
-			isInstanceTemplatePlacementTarget: {
+			isInstanceTemplatePlacementTarget: &schema.Schema{
 				Type:        schema.TypeList,
 				Computed:    true,
-				Description: "The placement restrictions to use for the virtual server instance.",
+				Description: "The placement restrictions for the virtual server instance. For the target tobe changed, the instance `status` must be `stopping` or `stopped`.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"id": {
+						"id": &schema.Schema{
 							Type:        schema.TypeString,
-							Computed:    true,
```

Comment:
Why is this changed to optional from computed?
isInstanceTemplatePlacementTarget is a computed variable

Reply:
similarly other